lib/puma/single.rb

```diff
@@ -19,6 +19,24 @@ def stats
       %Q!{ "backlog": #{b}, "running": #{r}, "pool_capacity": #{t}, "max_threads": #{m} }!
     end
 
+    def metrics
+      metrics = <<~METRICS
```

Comment:
Puma is currently testing with Ruby 2.2.x, and I believe `<<~` is only available with Ruby 2.3.x